stash_engine/app/controllers/stash_engine/resources_controller.rb

```diff
@@ -9,6 +9,10 @@ class ResourcesController < ApplicationController
 
     attr_writer :resource
 
+    def log
```

Comment:
I think you can leave defining this method out.  AFAIK, the `logger` is automatically exposed in standard Rails classes like any models, views or controllers.  For example, see https://guides.rubyonrails.org/debugging_rails_applications.html#the-logger for more info.  You just call `logger.<level>` like warn, error, info, etc and it should work.  You don't even need to write "Rails.logger" usually.

The places that I think you might see the logger called something else is in another library such as either a separate gem or else a non-rails-standard file where it isn't automatically exposed.  Sometimes it might not exist at all if a gem isn't used with Rails.  In other cases if something is always going to be used with Rails, it can likely be accessed by Rails.logger.

I'd argue that we should stick with calling it logger since it's the standard thing people are already used to in Rails.

I believe a lot of the weird logging is stuff David M wrote because he wasn't familiar with conventions, or external gems or me or someone else just being a little careless.

Reply:
Thanks! That's useful info. I'll change it.